README.md

```diff
@@ -27,36 +27,41 @@ Join us!
-- Know of anyone who might be interested? [Send them an email!](mailto:@?subject=Ten%20Simple%20Rules%20for%20Using%20Deep%20Learning%20in%20Biology%20paper&body=Take%20a%20look%20at%20the%20Deep%20Rules%20project%2C%20a%20PLOS%20Computational%20Biology%20paper%20written%20collaboratively%20using%20GitHub!%20%0A%0Ahttps%3A%2F%2Fgithub.com%2FBenjamin-Lee%2Fdeep-rules%0A%0AThe%20paper%2C%20entitled%20%22Ten%20Simple%20Rules%20for%20Using%20Deep%20Learning%20in%20Biology%2C%22%20is%20currently%20soliciting%20rule%20proposals%20from%20the%20community.%20To%20propose%20a%20rule%2C%20submit%20it%20here%3A%20https%3A%2F%2Fgithub.com%2FBenjamin-Lee%2Fdeep-rules%2Fissues%2Fnew%3Ftemplate%3Drule-proposal.md%0A%0AIf%20you%20know%20of%20anyone%20who%20might%20be%20interested%2C%20feel%20free%20to%20forward%20this%20email%20along!)
+- Help organize the [proposed rules](https://github.com/Benjamin-Lee/deep-rules/issues?q=label%3Arule) into one [document](https://github.com/Benjamin-Lee/deep-rules/blob/master/rules.md).
+- Know of anyone who might be interested? [Send them an email!](mailto:@?subject=Ten%20Simple%20Rules%20for%20Using%20Deep%20Learning%20in%20Biology%20paper&body=Take%20a%20look%20at%20the%20Deep%20Rules%20project%2C%20a%20PLOS%20Computational%20Biology%20paper%20written%20collaboratively%20using%20GitHub!%20%0A%0Ahttps%3A%2F%2Fgithub.com%2FBenjamin-Lee%2Fdeep-rules%0A%0AIf%20you%20know%20of%20anyone%20who%20might%20be%20interested%2C%20feel%20free%20to%20forward%20this%20email%20along!)
 - Have Twitter followers? [Tweet about the project!](https://twitter.com/intent/tweet?text=Have%20you%20used%20deep%20learning%20in%20your%20research%3F%20If%20so%2C%20contribute%20to%20the%20Ten%20Simple%20Rules%20for%20Deep%20Learning%20in%20Biology%20paper%21%0A%0Ahttp%3A%2F%2Fgithub.com%2FBenjamin-Lee%2Fdeep-rules)
 - Revise the [cover letter](https://github.com/Benjamin-Lee/deep-rules/edit/master/content/presubmission_inquiry.md)
+- Have an idea for a rule? [Propose it!](https://github.com/Benjamin-Lee/deep-rules/issues/new?template=rule-proposal.md)
 
-For more information, see [the contribution guidelines](CONTRIBUTING.md). All contributions are subject to the [code of conduct](CODE_OF_CONDUCT.md).
+For more information, see [the contribution guidelines](CONTRIBUTING.md). All
+contributions are subject to the [code of conduct](CODE_OF_CONDUCT.md).
 
 ## Manubot
 
 <!-- usage note: do not edit this section -->
 
-Manubot is a system for writing scholarly manuscripts via GitHub.
-Manubot automates citations and references, versions manuscripts using git, and enables collaborative writing via GitHub.
-The [Manubot Rootstock repository](https://git.io/vQSvo) is a general purpose template for creating new Manubot instances, as detailed in [`SETUP.md`](SETUP.md).
-See [`USAGE.md`](USAGE.md) for documentation how to write a manuscript.
+Manubot is a system for writing scholarly manuscripts via GitHub. Manubot
```

Comment:
Suggestion not to edit this section to avoid future conflicts when updating manubot.

Looks like the edits are mostly changing one line per sentence to word wrap... one sentence per line is highly recommended.

Reply:
Didn't know this could be an issue; reverted.